src/CMakeLists.txt

```diff
@@ -858,6 +853,10 @@ if(rocblas_FOUND)
     list(APPEND PACKAGE_STATIC_DEPENDS PACKAGE rocblas)
 endif()
 
+target_link_libraries( MIOpen INTERFACE $<BUILD_INTERFACE:hipblaslt> )
+target_link_libraries( MIOpen PRIVATE hipblaslt )
```

Comment:
That is not portable between OSes. Please use the imported target `roc::hipblaslt` instead of the library name.